cli/pkg/kctrl/cmd/kctrl.go

```diff
@@ -85,18 +86,27 @@ func NewKctrlCmd(o *KctrlOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Comm
 	cmd.AddCommand(NewVersionCmd(NewVersionOptions(o.ui, o.depsFactory), flagsFactory))
 
 	pkgCmd := cmdpkg.NewCmd()
+	pkgCmd.SetUsageTemplate(cobrautil.HelpSectionsUsageTemplate([]cobrautil.HelpSection{
+		cmdcore.PackageManagementCommandsHelpGroup,
+		cmdcore.PackageAuthoringCommandsHelpGroup,
+		cmdcore.PackageRepoHelpGroup,
+		cmdcore.RestOfCommandsHelpGroup,
+	}))
 	AddPackageCommands(o, pkgCmd, flagsFactory, pkgOpts)
 
 	cmd.AddCommand(pkgCmd)
 
 	appCmd := app.NewCmd()
+	appCmd.SetUsageTemplate(cobrautil.HelpSectionsUsageTemplate([]cobrautil.HelpSection{
+		cmdcore.AppManagementCommandsHelpGroup,
+		cmdcore.RestOfCommandsHelpGroup,
+	}))
 	appCmd.AddCommand(app.NewGetCmd(app.NewGetOptions(o.ui, o.depsFactory, o.logger), flagsFactory))
 	appCmd.AddCommand(app.NewListCmd(app.NewListOptions(o.ui, o.depsFactory, o.logger), flagsFactory))
 	appCmd.AddCommand(app.NewStatusCmd(app.NewStatusOptions(o.ui, o.depsFactory, o.logger), flagsFactory))
 	appCmd.AddCommand(app.NewPauseCmd(app.NewPauseOptions(o.ui, o.depsFactory, o.logger), flagsFactory))
 	appCmd.AddCommand(app.NewKickCmd(app.NewKickOptions(o.ui, o.depsFactory, o.logger), flagsFactory))
 	appCmd.AddCommand(app.NewDeleteCmd(app.NewDeleteOptions(o.ui, o.depsFactory, o.logger), flagsFactory))
-	appCmd.AddCommand(appinit.NewInitCmd(appinit.NewInitOptions(o.ui, o.depsFactory, o.logger)))
```

Comment:
Was this intentional?

Reply:
Yep `app init` is aimless without an `app release` command

Reply:
Hmm, yeah that makes sense, but I would have expected a different PR for this, or at least a different commit or description in the PR 😅 Right now it's as if we are trying to sneak in this change along with the main help section structuring 😆

Reply:
oooooof 😆 
That was not my intention, I was not sure if a line change would warrant another PR, happy to move this out if you think I should

Reply:
A different commit should be fine I guess.
But thinking more on this, we are actually removing a command which has been already released (not sure if people have started using it or not), but that still counts as a break change 🤔

Reply:
I do not see why anyone would, but then again I do not see any harm in leaving it dangling